app/views/trainees/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<h1 class="govuk-heading-xl">Trainee teacher data list</h1>
```

Comment:
Perhaps we should start storing the copy into `i18n` to make it easier to manage/change?

Reply:
I agree, but I don't think that should be part of this PR. We don't have this implemented yet and still working out that pages we need. I will be looking to create a page heading component so we dont need to have all this hardcoded html on every page.